### PR TITLE
Check for gzip by only reading first 3 bytes instead of fs.open

### PR DESCRIPTION
**Pull request by shehzan10**
Related to https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/issues/75 and https://github.com/AnalyticalGraphicsInc/3d-tiles-samples/pull/11#issuecomment-307770995

Once this is merged, I'll implement this in Cesium's server.js

### CODE REVIEW

**Comment by shehzan10** [comment]
bump @mramato or @lilleyse

File: server.js [hunk 1]
```diff
@@ -63,20 +63,19 @@
...
-                    res.header('Content-Encoding', 'gzip');
-                }
-                fs.close(fd, function(err) {
-                    next();
-                });
-            });
+        });
+
+        readStream.on('close', function() {
+            next();
```
Comment (mramato) [inline server.js]: Are we sure that `close` will always be called after `data`?  Couldn't we just call `next` at the end of `data` to be sure (and remove this event)?

Comment (shehzan10) [inline server.js]: Yeah I think that makes sense. Now that I think of it, this could also lead to double "next()" calls. I'll remove this.

Comment (shehzan10) [inline server.js]: Resolved.

**Comment by mramato** [comment]
Sorry if I'm missing something, but it looks like `next` is never called on successful read now.  Shouldn't there be one at the end of data?

**Comment by shehzan10** [comment]
> but it looks like next is never called on successful read now

Sorry about that. Fixed in the same commit.

**Comment by mramato** [comment]
Thanks!

**Pull request merged**